### PR TITLE
feat(osredirect):Redirect OSIO URLs to Landing page and update Header/Footer

### PR DESCRIPTION
**Pull request by abhinandan13jan**
@joshuawilson @christianvogt @nimishamukherjee Please review

### CODE REVIEW

**Comment by abhinandan13jan** [comment]
Updated Nginx to deny access to non index pages at root.

**Comment by christianvogt** [comment]
@abhinandan13jan I can still reach the old marketing pages with this nginx config.

Wouldn't it be easier to redirect all `*.html` references to the root `index.html`.That way you don't need to white list all the other file types?
Keep the fallback location changes to `index.html` though.

**Comment by abhinandan13jan** [comment]
@christianvogt 

- I've completely removed the individual flex-boxes instead of visibility hidden. 
- When I am redirecting all .html even with a fallback, it is experience an error with message, exceeded maximum number of redirects, i.e getting trapped in a loop.
-  After updating the marketing material as per 
https://docs.google.com/document/d/1vbn6l7ebXeRvBpMgfbrsOdZwMVpiVZvuxwMgOul3-Ww/edit#heading=h.eoiy3qil1tyh , this is what the Large screen and mobile screen view looks like.
- Updating the latest code to PR
Screenshot from 2019-04-01 19-59-53
Screenshot from 2019-04-01 20-00-04
Screenshot from 2019-04-01 20-00-16
Screenshot from 2019-04-01 20-00-24

**Comment by christianvogt** [comment]
@abhinandan13jan you may have gotten into a loop because you had the redirect of all `*.html` pages before the `index.html` location entry?

**Comment by christianvogt** [comment]
The footer looks squished in your 2nd screenshot.
The list items you created aren't actual lists. Maybe they should be?
We need someone from UX to review the look.

I'll try to run it locally to inspect the rest.

File: src/app/index.html [hunk 1]
```diff
@@ -100,46 +89,65 @@
...
+          <h2>Development Services on OpenShift.io</h2>
+          <div class="information-text text-left">
+            - Code with a Red Hat hosted in-browser IDE powered by Eclipse Che: 
+            Sign-up as SaaS |
+            Download CodeReady Workspaces for OpenShift |
+            Download Eclipse Che for any Kubernetes
+            <!--Inserting 3% height gap--><p class="design-gap"></p>
+            - Analyze your project’s dependencies using service that analyses millions of repositories using machine learning to check security vulnerabilities, licences and smart suggestion to improve your project: 
+            Use in <a href="https://che.openshift.io" target="blank">Hosted Che</a> |
+            <a href="https://marketplace.visualstudio.com/items?itemName=redhat.fabric8-analytics" target="blank">Add to VS Code</a> |
```
[the remark this answers is not in the document]
Comment (abhinandan13jan) [reply src/app/index.html]: updated

File: src/app/index.html [hunk 2]
```diff
@@ -100,46 +89,65 @@
...
+        <!--Intro Section Ends-->
+
+        <!--Inserting 3% height gap--><p class="design-gap"></p>
+
+        <!--2nd Section: Informative starts-->
+        <div class="register--content">
+          <h2>Development Services on OpenShift.io</h2>
+          <div class="information-text text-left">
+            - Code with a Red Hat hosted in-browser IDE powered by Eclipse Che: 
+            Sign-up as SaaS |
```
[the remark this answers is not in the document]
Comment (abhinandan13jan) [reply src/app/index.html]: No corresponding links found for this in https://docs.google.com/document/d/1vbn6l7ebXeRvBpMgfbrsOdZwMVpiVZvuxwMgOul3-Ww/edit#

File: src/app/index.html [hunk 3]
```diff
@@ -100,46 +89,65 @@
...
-      <div class="content--row_3 learn-more">
-        <a href="features.html">Learn more about the features <i class="fa fa-caret-right"></i></a>
-      </div>
-      <div class="content--row_4 openshiftio-chat">
-        <a href="https://chat.openshift.io/developers/channels/town-square" target="top" class="btn btn-bordered-blue"><i class="fa fa-comment"></i> Chat with us</a>
+        <!--2nd Section: Informative ends-->
+
+        <!--3rd Section: Informative starts-->
+        <div class="register--content">
+          <h2>Powered By Openshift</h2>
```
[the remark this answers is not in the document]
Comment (abhinandan13jan) [reply src/app/index.html]: updated

File: src/app/index.html [hunk 4]
```diff
@@ -100,46 +89,65 @@
...
+          <div class="information-text text-left">
+            Hosted Che is running on OpenShift - the Kubernetes platform for big ideas. 
+            OpenShift is the perfect target to run the services that you are developing with OpenShift.io. 
+            You can find more information about OpenShift here: <a href="https://www.openshift.com" target="blamk">https://www.openshift.com/</a>
```
[the remark this answers is not in the document]
Comment (abhinandan13jan) [reply src/app/index.html]: updated

**Comment by abhinandan13jan** [comment]
@christianvogt @joshuawilson @nimishamukherjee 
- - Quick Update,
- The changes made in this PR registers all the NGINX rules required for redirection and restricting access. 
- Requested components have been removed from the footer and the header
- The content section has been fed in with new marketing material as per the document https://docs.google.com/document/d/1vbn6l7ebXeRvBpMgfbrsOdZwMVpiVZvuxwMgOul3-Ww/edit#
and content has been center aligned. This is how things look now.
Screenshot from 2019-04-02 13-05-17
- Had a discussion with @Veethika from UX, we would need some layout changes, the footer looks imbalanced. 
We should merge this PR and take all the layout related changes in a separate PR.

File: src/app/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (christianvogt) [reply src/app/index.html]: @abhinandan13jan sign up link is the same as the other sign up link in the page. Che link I'm guessing is https://www.eclipse.org/che/ and the codeready workspace link i'm guessing is https://developers.redhat.com/products/codeready-workspaces/

[the remark this answers is not in the document]
Comment (abhinandan13jan) [reply src/app/index.html]: Updated the linksand the footer layout @christianvogt

**Comment by abhinandan13jan** [comment]
@christianvogt @serenamarie125 Footer updated. Looks something like this
Screenshot from 2019-04-08 13-54-43
Screenshot from 2019-04-08 13-55-09
Screenshot from 2019-04-08 13-55-20

**Pull request merged**